ceph-ansible-pr-syntax-check/build/build

```diff
@@ -19,3 +19,6 @@ mv infrastructure-playbooks/group_vars/all.yml.sample infrastructure-playbooks/g
 export ANSIBLE_LIBRARY=$WORKSPACE/ceph-ansible/library
 
 $VENV/ansible-playbook -i '127.0.0.1,' infrastructure-playbooks/*.yml --syntax-check --list-tasks -vv
+
+$VENV/ansible-lint site.yml
```

Comment:
I think you want ``site.yml.sample`` here?

Reply:
Yup, copy/paste too fast :), thanks!